muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/discussion/body/application/discussion-current-thread.tsx

```diff
@@ -140,32 +154,63 @@ class CurrentThread extends React.Component<
   /**
    * handles page changes,
    * sets selected page as currentPage to state
-   * @param event
    * @param selectedItem selectedItem
    * @param selectedItem.selected selected
    */
   handlePagerChange = (selectedItem: { selected: number }) =>
     this.getToPage(selectedItem.selected + 1);
 
+  /**
+   * handleSubscribeOrUnsubscribeClick
+   * @param thread thread
+   * @param isSubscribed isSubscribed
+   */
+  handleSubscribeOrUnsubscribeClick =
+    (thread: DiscussionThreadType, isSubscribed: boolean) =>
+    (e: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
```

Comment:
Why would you name an event and not use it in any way?

Reply:
Look lines 276 and 293